### PR TITLE
Made clearer the text pertaining to providing x and y to WGS-84

### PR DESCRIPTION
**Pull request by petraselmer**
Addresses [https://github.com/neo4j/neo4j/issues/11977](https://github.com/neo4j/neo4j/issues/11977)

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialFunctionsTest.scala [hunk 1]
```diff
@@ -133,7 +133,7 @@ class SpatialFunctionsTest extends DocumentingTest {
...
-      considerations("If any argument provided to `point()` is `null`, `null` will be returned.", "If the coordinates are specified using `latitude` and `longitude`, the `crs` or `srid` fields are optional and inferred to be `'WGS-84'` (srid=4326).", "If the coordinates are specified using `x` and `y`, then either the `crs` or `srid` field is required if a geographic CRS is desired.")
+      considerations("If any argument provided to `point()` is `null`, `null` will be returned.", "If the coordinates are specified using `latitude` and `longitude`, the `crs` or `srid` fields are optional and inferred to be `'WGS-84'` (srid=4326).", "If the coordinates are specified using `x` and `y`, the `crs` and `srid` fields are optional and inferred to be `'cartesian'` (srid=7203). If a geographic CRS is desired, then either the `crs` or `srid` field must be provided.")
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialFunctionsTest.scala]: The last sentence now feels redundant.

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialFunctionsTest.scala [hunk 2]
```diff
@@ -164,7 +164,7 @@ class SpatialFunctionsTest extends DocumentingTest {
...
-      considerations("If any argument provided to `point()` is `null`, `null` will be returned.", "If the `height/z` key and value is not provided, a 2D point in the _WGS 84_ CRS will be returned.", "If the coordinates are specified using `latitude` and `longitude`, the `crs` or `srid` fields are optional and inferred to be `'WGS-84-3D'` (srid=4979).", "If the coordinates are specified using `x` and `y`, then either the `crs` or `srid` field is required if a geographic CRS is desired.")
+      considerations("If any argument provided to `point()` is `null`, `null` will be returned.", "If the `height/z` key and value is not provided, a 2D point in the _WGS 84_ CRS will be returned.", "If the coordinates are specified using `latitude` and `longitude`, the `crs` or `srid` fields are optional and inferred to be `'WGS-84-3D'` (srid=4979).", "If the coordinates are specified using `x` and `y`, the `crs` and `srid` fields are optional and inferred to be `'cartesian'` (srid=7203). If a geographic CRS is desired then either the `crs` or `srid` field must be provided.")
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialFunctionsTest.scala]: The last sentence now feels redundant.

**Review by craigtaverner — CHANGES_REQUESTED** [review]

**Comment by petraselmer** [comment]
@craigtaverner addressed your review.

**Review by craigtaverner — CHANGES_REQUESTED** [review]
We started making the suggested changes and realised they actually do not work. These suggestions make sense if the considerations section was for a general section on the point function, which it is not. There are four sections, one for each of the CRS. The thinking is that each user would want to know how to make a point for their CRS of interest. The consideration sections are all different, each specific to the CRS under consideration, and written from that perspective. The asymmetry observed here is deliberate and correct.

**Comment by craigtaverner** [comment]
We started making the suggested changes and realised they actually do not work. These suggestions make sense if the considerations section was for a general section on the point function, which it is not. There are four sections, one for each of the CRS. The thinking is that each user would want to know how to make a point for their CRS of interest. The consideration sections are all different, each specific to the CRS under consideration, and written from that perspective. The asymmetry observed here is deliberate and correct.

**Pull request closed without merging**